python/darpinstances/instance.py

```diff
@@ -284,30 +284,34 @@ def read_instance(filepath: Path, travel_time_provider: MatrixTravelTimeProvider
         while (line_string):
             line = line_string.split()
             request_id: int = int(index)
-            request_time: int = int(line[0]) / 1000
+            request_timestamp: int = int(line[0]) / 1000
+            request_datetime = datetime.fromtimestamp(request_timestamp)
```

Comment:
all times should be read either from datetime string or timestamp, depending on what is present in the file